tests/src/lv_test_conf.h

```diff
@@ -32,7 +33,7 @@ extern "C" {
 #define LV_USE_BUILTIN_MALLOC   1
 #define LV_USE_BUILTIN_MEMCPY   1
 #define LV_USE_BUILTIN_SNPRINTF 1
-#define LV_MALLOC       lv_malloc_builtin
+#define LV_MALLOC       lv_malloc_test_wrapper
 #define LV_REALLOC      lv_realloc_builtin
```

Comment:
I'm not sure, but maybe we need this for realloc too. Or should realloc  return `lv_control_lv_malloc_stub` too?

Reply:
Not really sure, but it's easy to add, let me do that

Reply:
Done in [7cb4173](https://github.com/lvgl/lvgl/pull/4078/commits/7cb4173a5994c523908db70e3c982b02d934e4b4)

Reply:
Maybe we also need to stub free?